src/thunderbots/firmware/main/dribbler.c

```diff
@@ -14,19 +14,21 @@
 #include "physics.h"
 #include <stdio.h>
 
-#define SPEED_CONSTANT 3760.0f // rpm per volt—EC16 datasheet
+#define NOMINAL_SPEED_RPM 50800
+#define NOMINAL_VOLTAGE 18.0f
+#define SPEED_CONSTANT (NOMINAL_SPEED_RPM / NOMINAL_VOLTAGE) // rpm per volt
 #define VOLTS_PER_RPM (1.0f / SPEED_CONSTANT) // volts per rpm
 #define VOLTS_PER_RPT (VOLTS_PER_RPM * 60.0f * DRIBBLER_TICK_HZ) // volts per rpt, rpt=revolutions per tick
 #define VOLTS_PER_SPEED_UNIT (VOLTS_PER_RPT / 6.0f) // volts per Hall edge
-#define PHASE_RESISTANCE 0.403f // ohms—EC16 datasheet
+#define PHASE_RESISTANCE 0.512f // ohms—EC16 datasheet
 #define SWITCH_RESISTANCE (0.019f*2.0f) // ohms—AO4882 datasheet
 
-#define THERMAL_TIME_CONSTANT_WINDING 1.97f // seconds—EC16 datasheet
-#define THERMAL_RESISTANCE_WINDING 1.68f // kelvins per Watt—EC16 datasheet (winding to housing)
+#define THERMAL_TIME_CONSTANT_WINDING 2.17f // seconds—EC16 datasheet
```

Comment:
Maybe link a datasheet if possible here.

Reply:
@hannahvsawiuk 	for this one

Reply:
yee, I will grab the link

Reply:
https://www.maxongroup.com/medias/sys_master/root/8833419509790/19-EN-177.pdf

Reply:
Added datasheet link to top of constants section.